modules/sdk-lib-mpc/src/tss/ecdsa-dkls/util.ts

```diff
@@ -265,3 +274,77 @@ export async function generate2of2KeyShares(
   });
   return [partyA, partyB];
 }
+
+export function isGG18SigningMaterial(keyShare: string, walletPassphrase: string | undefined): boolean {
```

Comment:
Missing jsdoc.

Reply:
Addressed all your comments. Please review the changes